ops/col_importer.py

```diff
@@ -162,7 +171,8 @@ def add_to_scene(self, collection_prefix, link=True):
                                                            model.model_name),
                                            link
             )            
-            self.__add_spheres(collection, model.spheres)
+            self.__add_shapes(collection, model.spheres)
+            self.__add_shapes(collection, model.boxes, False)
```

Comment:
Might be better to add a new function at this point. Spheres and boxes code is different enough.

Reply:
Done.